esp32/modugfx.c

```diff
@@ -655,8 +638,6 @@ STATIC const mp_rom_map_elem_t ugfx_module_globals_table[] = {
     {MP_OBJ_NEW_QSTR(MP_QSTR_polygon), (mp_obj_t)&ugfx_polygon_obj},
     {MP_OBJ_NEW_QSTR(MP_QSTR_fill_polygon), (mp_obj_t)&ugfx_fill_polygon_obj},
 
-    {MP_OBJ_NEW_QSTR(MP_QSTR_print_fonts), (mp_obj_t)&ugfx_print_fonts_obj},
```

Comment:
are we implementing some API here or do we decide for ourselves what needs to be exposed?

 in any case the current implementation is ugly so removing the function altogether seems like an improvement to me, would appreciate a second set of eyes :)